go.mod

```diff
@@ -19,8 +21,8 @@ require (
 	go.uber.org/zap v1.13.0
 	golang.org/x/net v0.0.0-20210224082022-3d97a244fca7
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.20.2
-	k8s.io/apimachinery v0.20.2
-	k8s.io/client-go v0.20.2
+	k8s.io/api v0.21.1
+	k8s.io/apimachinery v0.21.1
+	k8s.io/client-go v0.21.1
```

Comment:
Once we upgrade the version of  `client-go`,  Please make sure what is the minimum k8s version we  can support, and modify the doc.

Reply:
OK.

Reply:
As per the compatibility matrix in https://github.com/kubernetes/client-go/tree/v0.21.1#compatibility-client-go---kubernetes-clusters, we should change the minimal supported k8s version to `1.15`.